src/shared/types.ts

```diff
@@ -10,7 +10,7 @@ export interface PullRequestInfo {
   // TODO: Move this into a function?
   dependencies?: PullRequestID[];
 
-  isOutdated: boolean;
+  isOutdated?: boolean; //NOTE: make isOutdated optional
```

Comment:
Nice!

We don't want to store these changelog-type comments in the code though, because these comments only make sense in the context of the PR, and once merged the comments won't make sense. You could always make review comments on your PRs as an alternative :)

```suggestion
  isOutdated?: boolean;
```